pkg/apiserver/handler_apis.go

```diff
@@ -48,7 +48,7 @@ func WithAPIs(handler http.Handler, informer informers.APIServiceInformer, servi
 }
 
 // apisHandler serves the `/apis` endpoint.
-// This is registered as a filter so that it never collides with any explictly registered endpoints
```

Comment:
Please open PRs against the source in kubernetes/kubernetes

The files here are mirrored from that repo